src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -490,7 +492,37 @@ public Script getScriptSigWithSignature(Script scriptSig, byte[] sigBytes, int i
      * a P2SH scriptSig.
      */
     public int getSigInsertionIndex(Sha256Hash hash, BtcECKey signingKey) {
```

Comment:
Should we rename the `hash` parameter maybe?

Reply:
Done